kotlin-react-redux/src/main/kotlin/react/redux/Imports.kt

```diff
@@ -6,9 +6,7 @@ package react.redux
 import react.*
 import redux.Store
 
-external class Provider : Component<ProviderProps, State> {
-    override fun render(): ReactElement?
-}
+external val Provider: ComponentClass<ProviderProps>
```

Comment:
This is not `ComponentClass`, it is function component in redux source code, so maybe it is better to declare it as `ComponentType`?

Reply:
Fixed